doc/source/formats/guessers/default.rst

```diff
@@ -0,0 +1,109 @@
+.. _default-guesser:
+
+==============
+DefaultGuesser
+==============
+
+
+.. warning::
+
+    The default guesser has been created to reproduce pre-v2.8.0 MDAnalysis guessing behaviour as much as possible. **Default behaviours will change in MDAnalysis version 3**, as detailed in deprecation warnings.
+
+
+The :class:`~MDAnalysis.guesser.default_guesser.DefaultGuesser` is the default guessing context for MDAnalysis. For historical reasons the ``DefaultGuesser`` largely works with biological conventions; for example, an atom named CA will be assumed to be a carbon rather than a calcium atom.
+
+
+Attributes guessed
+==================
+
+The topology attributes guessed by the default guesser are listed below, as are their dependencies and broad assumptions.
+Please see the `API documentation <https://docs.mdanalysis.org/stable/documentation_pages/guesser_modules/default_guesser.html>`_ for more details.
+
+.. _default-guesser-types:
+
+------------------
+Elements and types
+------------------
+
+The default guesser guesses atom ``element``\ s and ``type``\ s using the same pathway; when atom ``type``\ s are guessed, they represent the atom ``element``. Atom elements are guessed from the atom name. The default guesser follows biological naming conventions, where atoms named "CA" are much more likely to represent an alpha-carbon than a calcium atom. This guesser is still relatively fragile for non-traditionally biological atom names.
+
+The :meth:`~MDAnalysis.guesser.default_guesser.DefaultGuesser.guess_atom_element` method is used to guess atom elements or types following a process by which numbers, symbols, and some letters are stripped from the atom name and checked against a look-up table, as detailed in the linked API documentation. With this method, for example, "AO5*" would be guessed as "O", and "3hg2" as "H".
+
+
+------
+Masses
+------
+
+Masses are guessed by using a look-up table to get masses from the atom's ``element`` attribute. If ``element``\ s are not available, the atom's ``type`` is used in place of the element. If the ``type`` is not available, that is
+:ref:`guessed first <default-guesser-types>`.
+
+
+.. warning::
+
+    When an atom mass cannot be guessed from the atom ``type`` or ``name``, the atom is currently assigned a mass of 0.0.
+
+    Masses are guessed atom-by-atom, so even if most atoms have been guessed correctly, it is possible that some have been given masses of 0. It is important to check for non-zero masses before using methods that rely on them, such as :meth:`AtomGroup.center_of_mass`.
+
+
+.. important::
+
+    `np.nan` will be used as a default or "missing" value
+    in place of 0.0 for atom masses in version 3.0 of MDAnalysis.
+
+
+-------------
+Aromaticities
+-------------
+
+These are guessed using the :ref:`RDKit <RDKit-format>` converter by using the ``GetIsAromatic`` method.
+
```

Comment:
```suggestion

.. note:: 
   RDKit needs to have been installed for aromaticity guessing to be available.
   RDKit is always installed when MDAnalysis was installed with conda-forge packages
   but this may not be the case when using other installation paths.
```